laravel-api/api/User/Models/User.php

```diff
@@ -40,14 +42,16 @@ class User extends Model implements
      * @var array
      */
     protected $fillable = [
-        // 'domain_uuid',
+        'domain_uuid',
         'username',
         'password',
-        // 'contact_uuid',
+        'contact_uuid',
         'user_enabled',
         'add_user',
         'add_date',
         'user_email',
+        'salt',
+        'user_status'
```

Comment:
Check what this field means. Check FPBX docs and arrive at a decision who can update this status